src/wallet/test/wallet_tests.cpp

```diff
@@ -29,7 +29,7 @@ extern UniValue importmulti(const JSONRPCRequest& request);
 
 std::vector<std::unique_ptr<CWalletTx>> wtxn;
 
-typedef std::set<std::pair<const CWalletTx*,unsigned int> > CoinSet;
+typedef std::set<CInputCoin> CoinSet;
```

Comment:
Think about replacing old style "typedef  X Y " with the more intuitive C++11 "using Y = X" style